content/rs/administering/designing-production/networking/port-configurations.md

```diff
@@ -20,17 +20,17 @@ update your firewall with the port for that new database endpoint.
 | Protocol | Port | Description |
 |------------|-----------------|-----------------|
 | ICMP | * | For connectivity checking between nodes |
-| TCP | 1968 | Internal proxy usage. Ports are bound to loopback adapter. |
-| TCP | 3333, 3334, 3335, 3336, 3337, 3338, 3339, 36379, 36380 | Internal cluster usage |
-| TCP | 8001 | For application to access the RS [Discovery Service]({{< relref "/rs/concepts/data-access/discovery-service.md" >}}) |
-| TCP | 8443 | For secure (https) access to the management web UI |
-| TCP | 8444, 9080 | For nginx \<-> cnm_http/cm communications on the same host only. Ports are bound to loopback adapter. |
-| TCP | 9081 | For CRDB management |
+| TCP | 1968 | Proxy traffic (Internal use) |
```

Comment:
@bmansheim my original intent was to have a separate column to mark internal use, or even a separate table, but whatever works.

Reply:
If we could sort the table I would do that. Someday maybe.